charts/kafka-lag-exporter/Chart.yaml

```diff
@@ -2,6 +2,6 @@ apiVersion: v1
 appVersion: "0.5.5-SNAPSHOT"
 description: Kafka Lag Exporter
 name: kafka-lag-exporter
-version: 0.5.5
+version: 0.5.6
```

Comment:
This release process will do this automatically.  Can you revert this?

Reply:
reverted manual chart bump :+1: